src/ui/views/menu.coffee

```diff
@@ -432,6 +432,10 @@ templateWindow = (viewstate) -> [
 templateMenu = (viewstate) ->
     [
         {
+            label: ''
+        }
+        {
```

Comment:
can you make this a linux only fix?

I'd recommend that you create a variable at the top 

isLinux = platform == 'linux'

and then add a `if isLinux` after the '}' of that label